src/Workspaces/Core/Portable/Remote/WellKnownRemoteHostServices.cs

```diff
@@ -6,7 +6,7 @@ internal class WellKnownRemoteHostServices
     {
         public const string RemoteHostService = "remoteHostService";
         public const string RemoteHostService_Connect = "Connect";
-        public const string RemoteHostService_SynchronizeAsync = "SynchronizeAsync";
```

Comment:
See the work i did for my OOP features.  Where i define an interface and then use nameof in one place (where we need the string, and implement the interface in the other place to make sure hte name matches).